packages/wonder-blocks-modal/components/modal-launcher.js

```diff
@@ -74,7 +74,7 @@ type ControlledProps = {|
 
 type UncontrolledProps = {|
     ...CommonProps,
-    children: ({openModal: () => mixed}) => React.Node,
+    children: ({openModal: () => mixed, ...}) => React.Node,
```

Comment:
It feels like all these in this file should be strict, right? We're only passing in one argument. I see you're doing this in a number of files, any particular reason?

Reply:
Initially I thought that I'd keep the current types and just make things explicit.  I've had a change of heart and decided to make everything exact that could while making non-exact things explicit.